app/views/sales/current_month.html.erb

```diff
@@ -0,0 +1,31 @@
+<div class="sub-header">
+  <h3>Current Month's Sales for <%= @vendor.name %></h3>
+</div>
+<% total_sales = [] %>
+<% time_marker_month = Time.now.month %>
+<% time_marker_year = Time.now.year %>
+<table>
+  <% @sales.each do |s| %>
+    <% if s.purchase_time.month == time_marker_month && s.purchase_time.year == time_marker_year %>
```

Comment:
As awesome as it is to use something like `total_sales.inject(:+)` later on in this template, the determination of which sales are included in the current month should really live in the `Sale` model class.

Let me know on Slack and we can talk about optimal ways to achieve that.